pages/documentation/actions/index.ejs

```diff
@@ -0,0 +1,8 @@
+---
+title: Actions
+permalink: actions
+weight: 2
+---
+Here is the list of CBA Action Types:
+<%- site.listOfPages('actions'); %>
```

Comment:
The idea behind lately introduced changes, to prevent hardcoding current values, you can use here:
```
<%- site.listOfPages(page.pathname); %>
```

Same applies to other index pages, that yet uses hardcoded value